data/data-pipeline/data_pipeline/etl/constants.py

```diff
@@ -79,6 +69,16 @@
         "module_dir": "census_decennial",
         "class_name": "CensusDecennialETL",
     },
+    {
```

Comment:
Note: I moved these to the end of the list since they're slow and not used in the score. This way when you run `etl-run`, you'll fill in all the score-necessary data first.